recipe/meta.yaml

```diff
@@ -6,55 +6,61 @@ package:
   version: {{ version }}
 
 source:
-  url: https://cmake.org/files/v{{ major_minor }}/cmake-{{ version }}.tar.gz              # [not win]
-  fn: cmake-{{ version }}.tar.gz                                                          # [not win]
-  sha256: b3345c17609ea0f039960ef470aa099de9942135990930a57c14575aae884987                # [not win]
+  url: https://cmake.org/files/v{{ major_minor }}/cmake-{{ version }}.tar.gz
+  fn: cmake-{{ version }}.tar.gz
+  sha256: b3345c17609ea0f039960ef470aa099de9942135990930a57c14575aae884987
 
-  url: https://cmake.org/files/v{{ major_minor }}/cmake-{{ version }}-win32-x86.zip       # [win32]
-  fn: cmake-{{ version }}-win32-x86.zip                                                   # [win32]
-  sha256: dce666e897f95a88d3eed6cddd1faa3f44179d519b33ca6065b385bbc7072419                # [win32]
+  #url: https://cmake.org/files/v{{ major_minor }}/cmake-{{ version }}-win32-x86.zip       # [win32]
+  #fn: cmake-{{ version }}-win32-x86.zip                                                   # [win32]
+  #sha256: dce666e897f95a88d3eed6cddd1faa3f44179d519b33ca6065b385bbc7072419                # [win32]
 
-  url: https://cmake.org/files/v{{ major_minor }}/cmake-{{ version }}-win64-x64.zip       # [win64]
-  fn: cmake-{{ version }}-win64-x64.zip                                                   # [win64]
-  sha256: b530586b92241bc10aaaaa391bc9b8f1ae830a676050f9f5e6003d2ce6085abc                # [win64]
+  #url: https://cmake.org/files/v{{ major_minor }}/cmake-{{ version }}-win64-x64.zip       # [win64]
+  #fn: cmake-{{ version }}-win64-x64.zip                                                   # [win64]
+  #sha256: b530586b92241bc10aaaaa391bc9b8f1ae830a676050f9f5e6003d2ce6085abc                # [win64]
 
   patches:
     # Revert the whole commit in 3.11
-    - flang-fix.patch   # [win]
-    - cmake-3.10-boost-1.66-support.patch  # [not win]
+    - flang-fix.patch
+    - cmake-3.10-boost-1.66-support.patch
 
 build:
-  number: 2
+  skip: True  # [win and not py36]
+  number: 3
   detect_binary_files_with_prefix: true
+  features:
+    - vc14  # [win and py>=35]
```

Comment:
Might want to try `msvc_compiler` as we may need to build against a particular compiler, but we don't want to constrain where `cmake` can be installed as nothing is going to link to it. Things will only use `cmake` to build.

Reply:
Thanks, trying now.

Reply:
That is if you bundle your dependencies. But in this PR, dependencies are used from other packages. So, you need this

Reply:
Oh, maybe that's not what we want, maybe I should not use dependencies then.